View/Elements/shortcuts.ctp

```diff
@@ -64,5 +64,32 @@ Mousetrap.bind(['alt+j'], function(e) {
                 )); ?>";
         return false;
 });
+Mousetrap.bind(['alt+e'], function(e) {
+        location.href =
+                "<?php echo $this->Html->url(array(
+                        'plugin' => 'webzash',
+                        'controller' => 'entries',
+                        'action' => 'intex'
+                )); ?>";
+        return false;
+});
+Mousetrap.bind(['alt+a'], function(e) {
+        location.href =
+                "<?php echo $this->Html->url(array(
+                        'plugin' => 'webzash',
+                        'controller' => 'accounts',
+                        'action' => 'show'
+                )); ?>";
+        return false;
+});
+Mousetrap.bind(['alt+l'], function(e) {
```

Comment:
I question the usefulness of a hotkey to add a new ledger. Considering that most of the time you won't need it. I'd be quicker to add a new entry hotkey.